main.py

```diff
@@ -1 +1,22 @@
 #Mad Libs Generator Project
+loop = 1
+while (loop < 9):
+#All the questions that the program asks the user
+    noun = input("Choose a noun: ")
```

Comment:
Instead of repeating the input() method for each user input request - you can set up a for loop to reduce the amount of code here (lines 5- 10)